### ISSUE TITLE
i remove all pages please remove this page

### ISSUE THREAD

**Issue by mohitsolankioffically** [polkadot-js/phishing#1329]
i remove all pages please remove this page
![image](https://user-images.githubusercontent.com/102185721/159784749-32689e02-c9be-423d-918a-70869caac384.png)

**Comment by ccris02** [comment]
Just FYI as a quick workaround you can open the website in incognito mode or you could open it in IE or Edge which will not be blocked by the polkadot.js extension.

**Comment by jacogr** [comment]
Marking as duplicate of #1322 (same user, same site)
